examples/bert/requirements-gpu.txt

```diff
@@ -3,7 +3,6 @@ azure-identity
 datasets
 evaluate
 docker
-onnxruntime-gpu
```

Comment:
Since it is also an example for users, should we keep onnxruntime-gpu in this example requirements file ?

Reply:
Agree, updated in readme.md to guide users to select proper olive dependencies.